### PR TITLE
Clean up more language for inclusiveness

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
We had a number of references, mostly in docs, to the word master that can now be changed to main.  This PR does that and makes the project a bit more inclusive.

[NO NEW TESTS NEEDED]

Signed-off-by: tomsweeneyredhat <tsweeney@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/tutorials/basic_networking.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # Basic Networking Guide for Podman
 
 
-It seems once people master the basics of containers, networking is one of the first
```
Comment (rhatdan) [inline docs/tutorials/basic_networking.md:7]: This feels forced, but fine.

**Comment by rhatdan** [comment]
LGTM

File: docs/tutorials/basic_networking.md (hunk 1 above)
Comment (giuseppe) [reply docs/tutorials/basic_networking.md:7]: How is `conquer`more inclusive? :-)

I am not a native speaker so just ignore my note if you disagree, but the new text is not really clear.  Maybe "learn", "understand"?

Comment (rhatdan) [reply docs/tutorials/basic_networking.md:7]: I think understand is correct, None of us have really mastered or conquered it.

**Comment by vrothberg** [comment]
LGTM but CI isn't happy

File: docs/tutorials/basic_networking.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply docs/tutorials/basic_networking.md:7]: Changing to understand, good thought @giuseppe

**Comment by TomSweeneyRedHat** [comment]
I think I found the offending trailing whitespace and have nuked it.  I also touched up per Giuseppe's suggestion, I'll have to find something else to conquer now I guess.....

**Comment by TomSweeneyRedHat** [comment]
OK< I'm confuzeled.  This change is only md files with the exception of a dependebot script.  Yet I'm hitting:

```
[+1577s] [Fail] Podman play kube [It] podman play kube test with hostIPC 
[+1577s] /var/tmp/go/src/github.com/containers/podman/test/e2e/play_kube_test.go:4992
```

Is this a known issue?

**Comment by vrothberg** [comment]
> Is this a known issue?

Also happening in https://github.com/containers/podman/pull/17246. I am just going through emails but will catch up with what's going on here.

**Comment by vrothberg** [comment]
Also happening in https://github.com/containers/podman/pull/17228.

**Comment by vrothberg** [comment]
OK, it looks like commit 623ad2a63623 and https://github.com/containers/podman/pull/17186 got merged at the same time. They work in isolation but break in combination.

**Comment by vrothberg** [comment]
Please rebase https://github.com/containers/podman/pull/17247 got merged and the test will pass again.

**Comment by TomSweeneyRedHat** [comment]
Thanks for the sleuthing @vrothberg !  Rebased and repushed.

**Comment by TomSweeneyRedHat** [comment]
The "Test Bindings" test failed and left no trace of an error.  Restarted.  I dread these "straightforward" PRs like this.  I think the CI watches for them and goes into ultra-mischief mode....

**Pull request merged**